core-api/src/test/java/com/sequenceiq/cloudbreak/validation/UrlValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package com.sequenceiq.cloudbreak.validation;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Answers.RETURNS_DEEP_STUBS;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import javax.validation.ConstraintValidatorContext;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+public class UrlValidatorTest {
+
+    private UrlValidator underTest;
+
+    @Mock(answer = RETURNS_DEEP_STUBS)
+    private ConstraintValidatorContext context;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        when(context.buildConstraintViolationWithTemplate(any(String.class)).addConstraintViolation()).thenReturn(context);
+
+        underTest = new UrlValidator();
+    }
+
+    @Test
+    public void testValid() {
+        assertTrue(underTest.isValid("https://www.cloudera.com/", context));
```

Comment:
Might be an overkill, but a test could be added to check if local addresses are also allowed.

Reply:
I'll add that, since the validator is explicitly configured to permit them.